src/auto-trace.js

```diff
@@ -111,6 +149,10 @@ function executeAsyncMiddleware(asyncErr) {
  * @return {Error} the new error, after it has been passed through the middlewares
  */
 function executeSyncMiddleware(middlewares, ogErr) {
+	//Don't run middlewares on errors that have already been handled by auto-trace
+	if (ogErr && ogErr.autoTraceIgnore){
+		return ogErr
```

Comment:
Remind me why we do this inside of executeSyncMiddleware but not inside of executeAsyncMiddleware. I remember you explaining it but want to understand why.